package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-subsection-generator",
-  "version": "9.0.0",
+  "version": "10.0.0",
```

Comment:
This felt like a big enough change for this repo to make it a new major version since it could change how styles are applied and how to set up routes files to match for consistency.